src/getHumanAge.test.js

```diff
@@ -3,8 +3,104 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  // below are postive tests for EC and BV test designs
+  // including different values=)
+  test('check if 0, 0 inputs give 0,0 output', () => {
```

Comment:
'should return 0 for cat and dog when the input is 0'

Reply:
Agree. Fixed.